journal/updated/02-chapter-1.adoc

```diff
@@ -201,65 +204,66 @@ supposing you might be at the Quarterly Meeting.
 I hope you will not be negligent in attending such meetings,
 nor any other which are appointed for the service of Truth,
 and for the worship of Almighty God.
-I pray you, let not the indifferency of others lull you asleep;
+I beseech you, let not the indifferency of others lull you asleep,
 but be diligent and watchful, truly waiting in the depth of humility,
-and then I doubt not but you will be favoured with the presence of God,
+and then I doubt not that you will be favoured with the presence of God,
```

Comment:
I think this sounds better with both the **but** and the **that**:  and then I doubt not but that you will be favoured....[it's a little archaic, but it sounds kinda nice]